src/mongo/commands.ts

```diff
@@ -42,11 +42,18 @@ export class MongoCommands {
 	}
 
 	public static showResult(result: string, column?: vscode.ViewColumn): Thenable<void> {
-		let uri = vscode.Uri.file(path.join(vscode.workspace.rootPath, 'result.json'));
-		if (!fs.existsSync(uri.fsPath)) {
-			uri = uri.with({ scheme: 'untitled' });
+		let uri : vscode.Uri= null;
+		let openTextDocumentOptions : Object = {language : 'json'};
```

Comment:
'language' as 'json' doesn't help us in this case. We only register the 'Update' command when a file is named [result.json](https://github.com/Microsoft/vscode-cosmosdb/blob/master/package.json#L173) and the language doesn't matter.

For now I think we should just pop up an error message when a workspace isn't open. We can fix this case when we address this issue: https://github.com/Microsoft/vscode-cosmosdb/issues/13 (I think we should define a 'mongoresult' langauge type, but I don't want to do that before our first release)